classes/02class/exercises/c02-network10/mfreitassm/README.md

```diff
@@ -0,0 +1,49 @@
+# c02-network10
+
+## Commands Execution Output
+
+- Commands for creating an ENI with an IP from your private subnet
+
+```
+aws ec2 create-network-interface --subnet-id subnet-077d95081dbd28d26 --description "private network interface" --groups sg-04a749673e309f05d --private-ip-address 10.0.1.10
```

Comment:
About `--groups` (SGs):

A key take away here for any work you'll do in AWS: ENIs are the ones having SGs attached to. (there are a few exceptions)

When you create an EC2 instance, an elastic network interface will be created for it (ENI), and this ENI may have SGs associated. 

Besides that, ENIs are the ones having EIP (elastic IPs) attached to. 

So the info you get in the EC2 screen is basically the info from the ENI